graylog2-web-interface/src/components/configurations/CustomizationConfig.jsx

```diff
@@ -0,0 +1,153 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Badge, Button } from 'components/graylog';
+import { BootstrapModalForm, Input } from 'components/bootstrap';
+import { ColorPickerPopover, IfPermitted } from 'components/common';
+import ObjectUtils from 'util/ObjectUtils';
+import FormUtils from 'util/FormsUtils';
+import StringUtils from 'util/StringUtils';
+
+class CustomizationConfig extends React.Component {
+  static propTypes = {
+    config: PropTypes.shape({
+      badge_text: PropTypes.string,
+      badge_color: PropTypes.string,
+      badge_enable: PropTypes.bool,
+    }),
+    error: PropTypes.shape({
+      badge_text: PropTypes.string,
+    }),
+    updateConfig: PropTypes.func.isRequired,
+  };
+
+  static defaultProps = {
+    config: {
+      badge_text: 'PROD',
```

Comment:
Should the default be `DEV`?

Reply:
No, this is for use in Production where the user probably will have PROD or TEST and seldom DEV

Reply:
Already fixed here: https://github.com/Graylog2/graylog2-server/pull/5708/commits/8b34336dbecfd39a3cdafdb32a54895a3a361702#diff-510e99a3e4d1eff6038fd7db322a69b9R30